src/optimize/index.js

```diff
@@ -1,7 +1,23 @@
+import fs from 'fs';
 import FsOptimizer from './fs_optimizer';
+import i18nPlugin from '../plugins/i18n/server/i18n/index';
+import kibanaPackage from '../utils/package_json';
+import Promise from 'bluebird';
+
+const readdir = Promise.promisify(fs.readdir);
+
 module.exports = async (kbnServer, server, config) => {
   if (!config.get('optimize.enabled')) return;
 
+  server.log(
+    ['info', 'i18n'],
+    `Registering core plugin translations. This may take a few minutes`
+  );
+  let start = Date.now();
+  await registerCoreTranslations();
+  let seconds = ((Date.now() - start) / 1000).toFixed(2);
```

Comment:
minor, but the elapsed time logic seems like it would be worth putting in a function… it's used twice here and once in the lazy_optimizer.